firefox/lib/background.js

```diff
@@ -72,7 +72,7 @@ function init_variables(options) {
 
 function load_events() {
   pageMod.PageMod({
-    include: [/.*www\.google.*/],
+    include: [/\/\/www\.google/],
```

Comment:
`/\/\/(?:[a-z0-9-]+\.)*google(?:\.[a-z0-9-]+)+\//i` would work better.

This is because your `/\/\/www\.google/` also matches the website `www.google` on the [`.google` domain](https://domains.google/).